src/screens/SettingsScreen.js

```diff
@@ -0,0 +1,159 @@
+  Title,
+  TitleContainer,
+  TitleShadow,
+  Wrapper
+} from '../components';
+import { SettingsListItem } from '../components/SettingsListItem';
+import { arrowLeft } from '../icons';
+
+export const SettingsScreen = ({ navigation }) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const { orientation, dimensions } = useContext(OrientationContext);
+  const globalSettings = useContext(GlobalSettingsContext);
+  const [refreshing, setRefreshing] = useState(false);
+
+  const keyExtractor = (item, index) => `index${index}-id${item.id}`;
+
+  const renderSectionHeader = ({ section: { title } }) =>
+    !!title && (
+      <View>
+        <TitleContainer>
+          <Title accessibilityLabel={`${title} (Überschrift)`}>{title}</Title>
+        </TitleContainer>
+        {device.platform === 'ios' && <TitleShadow />}
+      </View>
+    );
+
+  const refresh = () => {
+    setRefreshing(true);
+    // TODO: do we need pull to refresh on the settings screen?
+    // we simulate state change of `refreshing` with setting it to `true` first and after
+    // a timeout to `false` again, which will result in a re-rendering of the screen.
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 500);
+  };
+
+  // TODO: get data from global settings
+  const { sections = {} } = globalSettings;
+
+  const sectionedData = [
+    {
+      data: [
+        {
+          title: texts.settingsTitles.pushNotifications,
+          topDivider: true,
+          type: 'toggle',
+          value: false
+        }
+      ]
+    },
+    {
+      data: [
+        {
+          title: texts.settingsTitles.analytics,
+          topDivider: true,
+          type: 'toggle',
+          value: true
+        }
+      ]
+    },
+    {
+      title: texts.settingsTitles.listLayouts.sectionTitle,
+      data: [
+        {
+          title: texts.settingsTitles.listLayouts.newsItemsTitle,
+          type: 'listLayout',
+          listSelection: 'Textliste',
+          Component: TextListItem
+        },
+        {
+          title: texts.settingsTitles.listLayouts.eventRecordsTitle,
+          type: 'listLayout',
+          listSelection: 'Textliste',
+          Component: TextListItem
+        },
+        {
+          title: texts.settingsTitles.listLayouts.toursTitle,
+          type: 'listLayout',
+          listSelection: 'Liste mit großen Bildern',
+          Component: CardListItem
+        },
+        {
+          title: texts.settingsTitles.listLayouts.pointsOfInterestTitle,
+          type: 'listLayout',
+          listSelection: 'Liste mit großen Bildern',
+          Component: CardListItem,
+          bottomDivider: true
+        }
+      ]
+    }
+  ];
+
+  return (
+    <SafeAreaViewFlex>
+      <SectionList
+        keyExtractor={keyExtractor}
+        sections={sectionedData}
+        renderItem={({ item, index, section }) => (
+          <SettingsListItem {...{ navigation, item, index, section, orientation, dimensions }} />
```

Comment:
we should split the logics in `SettingsListItem` into different components instead of checking for the `type` inside

Reply:
done in https://github.com/ikuseiGmbH/smart-village-app-app/pull/123/commits/34a9b3f9ec4b76f6aecdd7429316a4c3fbab4072